packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/network/dedicatedCloud-datacenter-network.controller.js

```diff
@@ -1,6 +1,55 @@
-export default class {
+import { DATACENTER_NETWORK_SITE_WEB_LINK } from './dedicatedCloud-datacenter-network.constants';
+import { NETWORK_LABEL } from '../../../../dedicatedCloud/datacenter/dedicatedCloud-datacenter.constants.js';
+
+export default class DedicatedCloudDatacenterNetworkTab {
   /* @ngInject */
-  constructor($translate) {
+  constructor($translate, ovhManagerPccDatacenterService, coreConfig) {
+    this.coreConfig = coreConfig;
     this.$translate = $translate;
+    this.ovhManagerPccDatacenterService = ovhManagerPccDatacenterService;
+
+    this.NETWORK_LABEL = NETWORK_LABEL;
+  }
+
+  $onInit() {
+    this.fetchNsxEdgeReference();
+    this.loadComsumptionOfOption();
+    const user = this.coreConfig.getUser();
+
+    this.ovhSubsidiary = user.ovhSubsidiary;
+
+    this.guideUrl =
+      DATACENTER_NETWORK_SITE_WEB_LINK[this.ovhSubsidiary] ||
+      DATACENTER_NETWORK_SITE_WEB_LINK.DEFAULT;
+  }
+
+  loadComsumptionOfOption() {
+    this.priceLoading = true;
+    return this.ovhManagerPccDatacenterService
+      .getConsumptionForecastByServiceId(this.nsxEdgeOptionServiceId)
+      .then((consumption) => {
+        this.consumption = consumption;
+      })
+      .finally(() => {
+        this.priceLoading = false;
+      });
```

Comment:
Define the property name related to the function name rather
```suggestion
    this.consumptionLoading = true;
    return this.ovhManagerPccDatacenterService
      .getConsumptionForecastByServiceId(this.nsxEdgeOptionServiceId)
      .then((consumption) => {
        this.consumption = consumption;
      })
      .finally(() => {
        this.consumptionLoading = false;
      });
```